src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt

```diff
@@ -239,7 +183,7 @@ class CourtCaseEventsListenerIntTest : IntegrationTestBase() {
       )
       .build()
     // when
-    val blitzer = Blitzer(100, 50)
+    val blitzer = Blitzer(100, 5)
```

Comment:
hopefully a bit more reliable but is easy to tweak to recreate the problem! 30 and 10 seem to fail a lot